ProcessMaker/Repositories/CaseParticipatedRepository.php

```diff
@@ -8,6 +8,11 @@
 
 class CaseParticipatedRepository
 {
+    /**
+     * @var CaseParticipated|null
```

Comment:
Please add a comment describing why reference to a $this->caseParticipated is needed

Reply:
Comment was added